acceptance/features/scaling.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Scaling down broker nodes
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Skipping incremental scale downs
```

Comment:
This test covers the same ground as `decommission` (which was behaviorally actually testing the GC decommissioning behavior previously since we didn't have controller over StatefulSet lifecycles), but goes farther and actually tests that we gracefully handle scaling down multiple nodes at once.